wg-checkpoint-restore/charter.md

```diff
@@ -0,0 +1,91 @@
+
+# WG Checkpoint Restore Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The Checkpoint/Restore Working Group aims to solve the problem of transparently
+checkpointing and restoring workloads in Kubernetes, a functionality discussed
+for over five years. The group will deliver the design and implementation of
+Checkpoint/Restore functionality in Kubernetes, serving as a central hub for
```

Comment:
Why it has to be a central part of Kubernetes, where multiple external solutions already exists?

Reply:
i personally like the idea of having it be integrated because then the ecosystem can rely on it. for instance, we could make eviction or preemption less disruptive in kubelet/kueue respectively

Reply:
> Why it has to be a central part of Kubernetes, where multiple external solutions already exists?

Can you be more specific about what already exists? Not sure what you are referring to?